sklearn/metrics/tests/test_score_objects.py

```diff
@@ -526,3 +536,27 @@ def test_scoring_is_not_metric():
                          Ridge(), r2_score)
     assert_raises_regexp(ValueError, 'make_scorer', check_scoring,
                          KMeans(), cluster_module.adjusted_rand_score)
+
+
+def test_copy_classes():
+    # Test the various properties of copy_classes parameter added
```

Comment:
you're not testing the case when ``copy_classes="something_else"`` and ``something_else`` is a valid parameter.